aspnetcore/security/key-vault-configuration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Azure Key Vault Configuration Provider | Microsoft Docs
+author: guardrex
+description: How to use the Azure Key Vault Configuration Provider to configure an application using name-value pairs loaded at runtime.
+keywords: ASP.NET Core, configuration, Azure Key Vault
+ms.author: riande
+manager: wpickett
+ms.date: 1/16/2017
+ms.topic: article
+ms.assetid: 0292bdae-b3ed-4637-bd67-19b9bb8b65cb
+ms.prod: aspnet-core
+uid: security/key-vault-configuration
+---
+# Azure Key Vault Configuration Provider
+
+By [Luke Latham](https://github.com/GuardRex) and [Andrew Stanton-Nurse](https://github.com/anurse)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/key-vault-configuration/sample)
+
+This document explains how to use the Azure Key Vault Configuration Provider to load application configuration values from Azure Key Vault. [Microsoft Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/) is a cloud-based service that helps you safeguard cryptographic keys and secrets used by applications and services.
+
+## When to use the provider
+Use the provider when your application has access to Azure services and requires encrypted storage of its configuration data. Common scenarios include:
+* Controlling access to sensitive configuration data
+* Monitoring and logging of configuration data use
+* Reducing latency, providing automatic scaling, and providing redundancy of configuration data
```

Comment:
> I kind'a like the Use this for section of these docs. Let me see if I can roll it into that paragraph.

Ultimately this is pretty simple:  Use the Key Vault config provider when you want to get secrets out of Azure Key Vault.  Why you want to use Key Vault is a separate question.